Modules/System/Email/src/Email/Outbox/EmailEditor.Page.al

```diff
@@ -369,13 +400,18 @@ page 13 "Email Editor"
         IsNewOutbox := true;
     end;
 
+    procedure GetEmailEditorId(): Guid
```

Comment:
This is achieved with Manual event Subscription. Instead of exposing an instance Id variable you should manually bind the event subscription on a specific codeunit instance on the OpenPage of your page extension, now you can set on this instance any other value you need, like contact, Invoice etc.